test/fit_path/__init__.py

```diff
@@ -0,0 +1,5 @@
+import sys
```

Comment:
I updated fit_path/__init__.py similar to @stuart-stanley's suggestion... and rebased.